service/service.py

```diff
@@ -219,22 +254,40 @@ def rename_wishlist(wishlist_id):
     return make_response(jsonify(wishlist.serialize()), status.HTTP_200_OK)
 
 ######################################################################
-# READ AN EXISTING ITEM FROM WISHLIST
+# PATH: /wishlists{id}/items/{id}
```

Comment:
the path should be `/wishlists/{id}/items/{id}`

Reply:
It was a typo! Corrected!